Server/mods/deathmatch/logic/packets/CPlayerListPacket.cpp

```diff
@@ -131,20 +131,20 @@ bool CPlayerListPacket::Write(NetBitStreamInterface& BitStream) const
         BitStream.WriteBit(pPlayer->IsFrozen());
 
         // Nametag stuff
-        unsigned char ucNametagTextLength = 0;
-        char*         szNametagText = pPlayer->GetNametagText();
-        if (szNametagText)
-            ucNametagTextLength = static_cast<unsigned char>(strlen(szNametagText));
+        unsigned char    ucNametagTextLength = 0u;
+        std::string_view nametagText = pPlayer->GetNametagText();
+
+        ucNametagTextLength = (unsigned char)nametagText.length();
 
         if (!BitStream.Can(eBitStreamVersion::UnicodeNametags))
```

Comment:
Is this even a 1.5 version? Otherwise we could drop this

Reply:
The unicode stuff?

Reply:
yup

Reply:
```cpp
//
// 1.5.3 RELEASED - 2016-10-20
//

// Fix player nametag unicode characters missing on player join
// 2016-12-09 0x067 2e582453b476c1183bd9fae5363a7cffdb531834
 ```
 It's pretty recent actually.